src/bttrx_fsm.h

```diff
@@ -22,12 +22,14 @@ Contact: bt-trx.com, mail@bt-trx.com
 
 #ifdef ARDUINO
 #include <Arduino.h>
+#include "bttrx_display.h"
```

Comment:
Duplicates the include in Line 32. This line can be removed, as bttrx_display.h has its own #ARDUINO include guard.